systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -145,4 +148,28 @@ public void testKafkaConnectScaleUpScaleDown() {
             assertThat(events, hasNoneOfReasons(Failed, Unhealthy, FailedSync, FailedValidation));
         }
     }
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectCluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG)
+    public void testForUpdateValuesInConnectCM() {
+        List<String> connectPods = kubeClient.listResourcesByLabel("pod", "strimzi.io/type=kafka-connect");
+        replaceCm(CONNECT_CLUSTER_NAME, "healthcheck-delay", "120");
+        replaceCm(CONNECT_CLUSTER_NAME, "healthcheck-timeout", "10");
```

Comment:
The `replaceCM` will patch AFAIK the ConfigMap. By calling it twice, you are most probably doing two separate updates which trigger two rolling updates. Again the test might take much longer than necessary because of this. Yo should do all changes within one update as done for example here: https://github.com/strimzi/strimzi/blob/4f56854df3ec75e5fc6d53e9325b0adb7442cd7d/systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java#L254